third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -890,6 +890,26 @@ func TestAccContainerCluster_withSandboxConfig(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"min_master_version"},
 			},
+			{
+				// GKE sets automatic labels and taints on nodes. This makes
+				// sure we ignore the automatic ones and keep our own.
+				Config:   testAccContainerCluster_withSandboxConfig(clusterName),
+				PlanOnly: true,
```

Comment:
Can you also add a comment here about what this does, since it's a less common feature in our tests?

Reply:
Done!